apps/src/applab/reducers.js

```diff
@@ -48,7 +48,10 @@ function level(state, action) {
         'instructionsMarkdown',
         'instructionsInTopPane',
         'puzzleNumber',
-        'stageTotal'
+        'stageTotal',
+        'showDebugButtons',
+        'showDebugConsole',
+        'showDebugWatch',
```

Comment:
The initialLevelProps reducers in applab and gamelab have a lot in common. Wonder if we want to share that code in some way eventually.